src/models.ts

```diff
@@ -139,6 +151,11 @@ export interface IFilterMeasureTarget extends IBaseFilterTarget {
 
 export declare type IFilterTarget = (IFilterColumnTarget | IFilterHierarchyTarget | IFilterMeasureTarget);
 
+export interface IFiltersContainer {
+  target?: ITarget,
```

Comment:
We already have IBasicFilter and IAdvancedFilter, adding ILevelFilter makes it sound like it's another type of filter.  I think anything with Filter suffix should be a filter object.

I think ILevel, IPageLevel, IVisualLevel, or IScope, IPageScope, IVisualScope makes it more clear they are not interfaces for filters.